docs/sections/user_guide/cli/mode_template.rst

```diff
@@ -273,7 +273,7 @@ The examples in this section use atparse-formatted template file ``atparse.txt``
   .. code-block:: text
 
      $ uw template translate --input-file atparse.txt
-     {{greeting}}, {{recipient}}!
+     {{ greeting }}, {{ recipient }}!
```

Comment:
Jinja doesn't technically care about the spaces in expressions, but the official documentation always includes them,  so this PR follows their lead.